frontend/src/components/dialogs/GameDetailsDialog.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {Dialog, DialogContent, DialogTitle, Stack, Typography} from '@mui/material';
+import {useDialog} from './useDialog.ts';
+import {default as _ReactPlayer} from 'react-player/lazy';
+import {ReactPlayerProps} from "react-player/types/lib";
+
+const ReactPlayer = _ReactPlayer as unknown as React.FC<ReactPlayerProps>;
```

Comment:
Warum musst du hier so einen krassen Type cast machen?

Reply:
Nach ein bisschen Recherche erinnere ich mich wieder: ich hatte in meinem Typescript/Vite Projekt Probleme mit ReactPlayer und übernahm dann folgende Lösung: https://github.com/cookpete/react-player/issues/1406#issuecomment-1179666900

Das funktioniert, erzeugte aber diesen hässlichen Code.

Die Lösung zwei Kommentare weiter ist deutlich besser, hier muss ich lediglich @types/react installieren, was vermutlich sowieso sinnvoll ist.